ros2run/ros2run/command/run.py

```diff
@@ -56,4 +60,5 @@ def main(self, *, parser, args):
             raise RuntimeError(msg)
         if path is None:
             return 'No executable found'
-        return run_executable(path=path, argv=args.argv)
+        prefix=args.prefix[0] if args.prefix is not None else ''
```

Comment:
The argument should either be a single argument wrapped in quotes or accept multiple arguments separated by spaces. In the first case calling `str.split` isn't enough since the argument might contained quotes substrings. Therefore you will need to use `shlex.split` instead.